scripts/bootloader/provision.py

```diff
@@ -13,13 +13,19 @@
 from hashlib import sha256
 
 
-def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output):
+def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output, max_size=0x1000,
```

Comment:
To make this a bit more visible to the user, would it make sense to not have a default value here, but instead have a default value for the argument `--max-size` ?

Reply:
Done